specs/operator/auto-prune.spec

```diff
@@ -412,4 +412,34 @@ Steps:
   * Assert pruner cronjob(s) in namespace "target namespace" contains "1" number of container(s)
   * Delete project "test-project-1"
   * Delete project "test-project-2"
-  * Remove auto pruner configuration from config CR
\ No newline at end of file
+  * Remove auto pruner configuration from config CR
+
+## Verify that the operator is up and running after deleting namespace with pruner annotation: PIPELINES-12-TC15
+Tags: e2e, integration, auto-prune, admin, cronjob, sanity
```

Comment:
We have too many scenarios with `sanity` tag in this spec... we should limit their number significantly. I suggest to have just 1 or 2, e.g. the most basic one and the most complex one.

Reply:
Will remove it from few tests after testing this particular test case

Reply:
:+1: